vyked/bus.py

```diff
@@ -246,6 +249,24 @@ def _handle_blacklist(self, protocol):
         self.pubsub._is_blacklisted = True
         protocol.send('Service Blacklisted Successfully')
 
+    def send_http_request(self, app: str, service: str, version: str, method: str, entity: str, params: dict):
+        host, port, node_id, service_type = self._registry_client.resolve(service, version, entity, HTTP)
+        url = 'http://{}:{}{}'.format(host, port, params.pop('path'))
+
+        http_keys = ['data', 'headers', 'cookies', 'auth', 'allow_redirects', 'compress', 'chunked']
+        kwargs = {k: params[k] for k in http_keys if k in params}
+
+        query_params = params.pop('params', {})
+
+        if app is not None:
+            query_params['app'] = app
+
+        query_params['version'] = version
+        query_params['service'] = service
+        response = yield from asyncio.wait_for(asyncio.shield(self._aiohttp_session.request(method, url, params=query_params, **kwargs)), 60)
```

Comment:
append /\_\_onemg-internal\_\_ to url so that every dev does not need to explicitly mention that in the url, we can have the logic to control this param, and can remove depending upon our need.
Moreover this method to be only used for inter-service communication in which is_internal is True

Reply:
can we add a sample file of how to use this

Reply:
added support to add prefix to each http call using config file .

also added sample code in readme file